src/styles/generator.module.css

```diff
@@ -0,0 +1,125 @@
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--background);
+  max-width: 100vw;
+  height: 100vh;
+  /* background-image: url("/telAviv.webp");
+    background-size: cover;
+    background-repeat: no-repeat;
+    position: relative; */
```

Comment:
Delete this comment if it's not needed or useful, if it is add a explaination :)